### PR TITLE
add YetAnotherCompany projects to starknet.toml

### PR DESCRIPTION
**Pull request by rcatalan98**

### CODE REVIEW

File: data/ecosystems/s/starknet.toml [hunk 1]
```diff
@@ -3290,9 +3291,6 @@ url = "https://github.com/jejemerong/triality"
 [[repo]]
 url = "https://github.com/JejomStark/SymbOnStark"
 
-[[repo]]
-url = "https://github.com/JejomStark/ShellOnStark"
```
Comment (eherrerosj) [inline data/ecosystems/s/starknet.toml:3294]: why remove?

File: data/ecosystems/s/starknet.toml [hunk 2]
```diff
@@ -5134,9 +5132,6 @@ url = "https://github.com/ProjectKepler1/imx-starknet"
 [[repo]]
 url = "https://github.com/ProNodes11/Starknet-checker"
 
-[[repo]]
-url = "https://github.com/Prop-House/prop-house-monorepo"
```
Comment (eherrerosj) [inline data/ecosystems/s/starknet.toml:5138]: why remove?

File: data/ecosystems/s/starknet.toml [hunk 3]
```diff
@@ -7359,6 +7354,9 @@ url = "https://github.com/yasmeen/starknet-app"
 [[repo]]
 url = "https://github.com/yekssin/starknet"
 
+[[repo]]
+url = "https://github.com/yetanotherco/yet-another-bridge"
```
Comment (eherrerosj) [inline data/ecosystems/s/starknet.toml:7358]: no need to add if you are creating a new ecosystem that includes this URL and adding such eco as a sub-eco of this eco

File: data/ecosystems/s/starknet.toml [hunk 4]
```diff
@@ -21,6 +21,7 @@ sub_ecosystems = [
   "Playmint",
   "StarkWare",
   "ViewBlock",
+  "Yet Another Company"
```
Comment (eherrerosj) [inline data/ecosystems/s/starknet.toml:24]: missing toml

**Review by eherrerosj — CHANGES_REQUESTED** [review]
You need to create a new ecosystem for Yet Another Company. For that, you need to create a new toml file like yet-another-company.toml with title "Yet Another Company" and add the org in it (https://github.com/yetanotherco)

**Comment by rcatalan98** [comment]
@eherrerosj your comments are already addressed thank you for taking the time to review it

**Pull request merged**